packages/components/src/button/README.md

```diff
@@ -128,7 +128,7 @@ Name | Type | Default | Description
 --- | --- | --- | ---
 `disabled` | `bool` | `false` | Whether the button is disabled. If `true`, this will force a `button` element to be rendered.
 `href` | `string` | `undefined` | If provided, renders `a` instead of `button`.
-`isSecondary` | `bool` | `false` | Renders a default button style.
+`isDefault` | `bool` | `false` | Renders a default button style.
```

Comment:
Thanks for the PR. 

isSecondary is still the correct prop here. That's because it's not released yet, the change should be effective in the next release

Reply:
OK. Got it. 
What about isDefault? It still applies the class(using WP 5.3).

Reply:
It will be supported but deprecated in the future. You can just keep using it for the moment.